gcalbot/gcalbot/db.go

```diff
@@ -0,0 +1,113 @@
+package gcalbot
+
+import (
+	"database/sql"
+	"time"
+
+	"github.com/keybase/managed-bots/base"
+
+	"golang.org/x/oauth2"
+)
+
+type DB struct {
+	*base.DB
+}
+
+func NewDB(db *sql.DB) *DB {
+	return &DB{
+		DB: base.NewDB(db),
+	}
+}
+
+func (d *DB) GetToken(identifier string) (*oauth2.Token, error) {
```

Comment:
Maybe this can move to a `GoogleOAuthDB` shared amongst the google bots

Reply:
Sounds good. cc @Daniel-Wang